core/src/test/java/io/questdb/test/client/LineSenderBuilderTest.java

```diff
@@ -605,6 +604,18 @@ public void testDnsResolutionFail() throws Exception {
         });
     }
 
+    @Test
+    public void testDuplicatedAddresses() throws Exception {
```

Comment:
Could you add a test to verify that duplicate hosts, but different ports are allowed?

Reply:
done